api/scpca_portal/views/computed_file.py

```diff
@@ -88,6 +89,7 @@ def get_serializer_context(self):
                     {"message": "Your token is not valid or not activated.", "token_id": token_id}
                 )
 
+            TokenDownload.track(token_id, computed_file_pk)
```

Comment:
I would move this here for clarity, but it's not essential.

```suggestion
            computed_file_pk = self.kwargs.get("pk")
            TokenDownload.track(token_id, computed_file_pk)
```